CONTRIBUTING.md

```diff
@@ -0,0 +1,74 @@
+
+![](https://i.imgur.com/fAJEFWH.png)
+
+If you find an existing bug report, add a description of your experience to that issue.
+
+If you don’t find an existing bug report, create a new issue in the appropriate repository (see list of Nextstrain components above) and select "Bug report."
+
+## Request a feature
+
+If you’ve thought of a way to improve Nextstrain, first [search our existing issues](https://github.com/search?q=org%3Anextstrain&state=open&type=Issues) to see if someone else already requested the same thing.
+
+If you don't find an existing feature, create a new issue in the appropriate repository (see list of Nextstrain components above) and select "Feature request."
+
+## Contribute documentation
+
+Did you find a typo or a mistake in our existing documentation? Did you work through an example in a tutorial only to find it was missing a step? Are you trying to accomplish specific something with Nextstrain but can't find any description of that analysis? Did you find [an open issue on our project boards](https://github.com/orgs/nextstrain/projects) for improving documentation and you want to help out?
+
+These are all great opportunities for you to help improve our documentation. Find the Nextstrain component you'd like to contribute and review the contributing guide for that component's documentation using the links below.
+
+  - [nextstrain.org](https://nextstrain.org/docs/contributing/documentation)
+  - [auspice](https://github.com/nextstrain/auspice/tree/master/docs-src)
+  - [augur](https://github.com/nextstrain/augur/#documentation)
+
+
+# Contribute science & communications  
+
+Nextstrain is more than the software used to build and visualize data on nextstrain.org. We rely on contributions from other scientists including data, scientific interpretation of results, and language translations of situation reports, user interfaces, and documentation. 
+
+## Submitting data 
+Please refer to the [pathogen-specific repository](https://github.com/nextstrain) for information about where data is being sourced from. For example, the SARS-CoV-2 data is being pulled from GISAID; we'll pick up any new sequence data submitted there and include it in our next public build. 
+
+Alternatively, you may be interested in making your own [community build](https://nextstrain.org/docs/contributing/community-builds) for your specific dataset.
+
+## Sharing interpretations  
+Nextstrain Narratives allow scientists to easily share text linked to corresponding Nextstrain visuals. For example, we use these for our COVID-19 Situation Reports. You can [learn more about Narratives here](https://nextstrain.org/docs/narratives/introduction).
+
+## Foreign language translations  
+Please see [this project board]() for more information about how to help with foreign language translations.
```

Comment:
I'm not sure if Cassia is interested in using a project board to manage translation issues, so I'll follow up with her about this today. If we don't use a project board, could we link to hello@nextstrain.org here instead?